crates/levm/src/operations.rs

```diff
@@ -187,3 +211,76 @@ impl Operation {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn push0_correct_bytecode() {
+        let op = Operation::Push0.to_bytecode();
+        assert_eq!(op, Bytes::from(vec![0x5f]))
+    }
+
+    #[test]
+    fn push1_correct_bytecode() {
+        let op = Operation::Push((1, 0xff.into())).to_bytecode();
+        assert_eq!(op, Bytes::from(vec![0x60, 0xff]))
+    }
+
+    #[test]
+    fn push31_correct_bytecode() {
```

Comment:
```suggestion
    fn push32_correct_bytecode() {
```

or maybe change the test to use a `push((31, U256::MAX))`

Reply:
You're right! I corrected this to have one for push31 and another for push32.